auction-deploy/auction_deploy/core.py

```diff
@@ -170,3 +172,66 @@ def get_deployed_auction_contracts(
     )
 
     return deployed_auction_contracts
+
+
+def whitelist_addresses(
+    auction_contract: Contract,
+    whitelist: Sequence[str],
+    *,
+    batch_size=100,
```

Comment:
I don't like having a default here since you already have one in the cli

Reply:
ok